nasa-gibs/worldview
# Add distraction free mode query string param, add shortcuts section in about page

## Description

Fixes #3391  .

- [x] Add distraction free mode query string parameter `df` activated as `df=true` and `false` by default. This enhancement is accompanied by a timed out alert when distraction free mode is initially activated and/or a page is navigated to with the feature active.
- [x] Add keyboard shortcuts section to about page that is only visible on app mobile breakpoint width. Revise about page css, remove unused pages files. and move styling out of pages section.
- [x] Revise toolbar button to use separate "exit distraction free mode" button and remove reuse of info button.

## How To Test

1. Open with this URL parameter `/?df=true` to see query string loaded distraction free mode and alert dialog
2. Open about page in desktop width to see keyboard shortcuts

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview
